### ISSUE TITLE
Missing aria-label in Help

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#3959]
Discovered by @yileifeng 

Console error: missing aria label when opening help 
- add attribute to line 24 in file help-summary.html
